presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4086,6 +4087,19 @@ public void testTemporaryStagingDirectorySessionProperties()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testLikeSerializes()
+    {
+        Session pushdownFilterEnabled = Session.builder(getQueryRunner().getDefaultSession())
+                .setCatalogSessionProperty(HIVE_CATALOG, PUSHDOWN_FILTER_ENABLED, "true")
+                .build();
+        assertUpdate("CREATE TABLE test_create (a varchar)");
```

Comment:
- use `lineitem.comment` instead:

```
        Session pushdownFilterEnabled = Session.builder(getQueryRunner().getDefaultSession())
                .setCatalogSessionProperty(HIVE_CATALOG, PUSHDOWN_FILTER_ENABLED, "true")
                .build();
        assertQuerySucceeds(pushdownFilterEnabled, "SELECT comment from lineitem where comment LIKE 'abc%'");
```

- I'm thinking that the test name may need to be updates to clarify that this is pushdown-filter-specific test.